README.md

```diff
@@ -33,6 +33,10 @@ Here are some of the benefits of using this library:
 * [Targeting](#targeting)
   * [Targeting Exclusion](#targeting-exclusion)
 * [Variants](#variants)
+* [Telemetry](#telemetry)
+    * [Application Insights Telemetry Publisher](#application-insights-telemetry-publisher)
+    * [Custom Telemetry Publishers](#custom-telemetry-publishers)

```

Comment:
I suggest swapping the order of "Appliction Insights Telemetry Publisher" and "Custom Telemetry Publishers" sections.
The reason is that it will be more consistent with the feature filter section where we introduce how to implement feature filters before introducing the built-in filters.

Reply:
+1 for the suggestion. From a reader perspective, it's easier to comprehend the concepts by following the order of how to enable telemetry, what is telemetry publisher, and how to use an out-of-box publisher. Suggest the following order/title:

* Enabling Telemetry
* Telemetry Publishers
* Application Insights Telemetry Publisher

Reply:
sounds good. will update.

Reply:
Updated, thanks for the suggestion.